media-source-respec.html

```diff
@@ -467,51 +467,51 @@ <h3>Attributes</h3>
                     </p>
                     </li>
                   </ol>
                 </li>
               </ol>
             </li>
 
-            <li>If <var>sourceBuffer</var> is in {{MediaSource/activeSourceBuffers}}, then remove <var>sourceBuffer</var> from {{MediaSource/activeSourceBuffers}} and
+            <li>If |sourceBuffer| is in {{MediaSource/activeSourceBuffers}}, then remove |sourceBuffer| from {{MediaSource/activeSourceBuffers}} and
               [=queue a task=] to [=fire an event=] named {{removesourcebuffer}} at the <a>SourceBufferList</a> returned by {{MediaSource/activeSourceBuffers}}.</li>
-            <li>Remove <var>sourceBuffer</var> from {{MediaSource/sourceBuffers}} and [=queue a task=] to [=fire an event=] named {{removesourcebuffer}} at
+            <li>Remove |sourceBuffer| from {{MediaSource/sourceBuffers}} and [=queue a task=] to [=fire an event=] named {{removesourcebuffer}} at
               the <a>SourceBufferList</a> returned by {{MediaSource/sourceBuffers}}.</li>
-            <li>Destroy all resources for <var>sourceBuffer</var>.</li>
+            <li>Destroy all resources for |sourceBuffer|.</li>
           </ol>
-        <table class="parameters"><tbody><tr><th>Parameter</th><th>Type</th><th>Nullable</th><th>Optional</th><th>Description</th></tr><tr><td class="prmName">sourceBuffer</td><td class="prmType">{{SourceBuffer}}</td><td class="prmNullFalse"><span role="img" aria-label="False">✘</span></td><td class="prmOptFalse"><span role="img" aria-label="False">✘</span></td><td class="prmDesc"></td></tr></tbody></table><div><em>Return type: </em>{{undefined}}</div></dd><dt><dfn><code>endOfStream</code></dfn></dt><dd>
+        <table class="parameters"><tbody><tr><th>Parameter</th><th>Type</th><th>Nullable</th><th>Optional</th><th>Description</th></tr><tr><td class="prmName">|sourceBuffer|</td><td class="prmType">{{SourceBuffer}}</td><td class="prmNullFalse"><span role="img" aria-label="False">✘</span></td><td class="prmOptFalse"><span role="img" aria-label="False">✘</span></td><td class="prmDesc"></td></tr></tbody></table><div><em>Return type: </em>{{undefined}}</div></dd><dt><dfn><code>endOfStream</code></dfn></dt><dd>
           <p>Signals the end of the stream.</p>
 
           <ol class="method-algorithm">
             <li>If the {{MediaSource/readyState}} attribute is not in the {{ReadyState/""open""}} state then throw an {{InvalidStateError}} exception and abort these steps.</li>
             <li>If the {{SourceBuffer/updating}} attribute equals true on any <a>SourceBuffer</a> in {{MediaSource/sourceBuffers}}, then throw an {{InvalidStateError}} exception and abort these steps.</li>
-            <li>Run the [=end of stream=] algorithm with the <var>error</var> parameter set to <var>error</var>.</li>
+            <li>Run the [=end of stream=] algorithm with the |error:EndOfStreamError| parameter set to |error|.</li>
```

Comment:
I note that ReSpec gets slightly confused here as it thinks that both occurrences of `error` refer to the same variable, whereas the first `|error:EndOfStreamError|` actually refers to the name of the parameter in the end of stream algorithm, whereas the second `|error|` refers to the name of the parameter of the `endOfStream` method that is being described here.

At some point in the future, there may exist a way to flag algorithms and algorithm parameters as such (see related discussion in https://github.com/speced/spec-dfn-contract/issues/2), which would provide a way to distinguish between the two. I don't have a good workaround to suggest in the meantime other than to drop the `|` around the first occurrence (to end up with an undecorated "with the error parameter"). This does not strike me as very confusing in any case, so leaving things as-is seems good too.